### PR TITLE
First capstone project: Featured branch

### PR DESCRIPTION
**Pull request by Megagig**
### In This Milestone, I: 

- Creating a new repository for the project

-  Creating a new branch (Mobile-Desktop-version) for the project

-  Checking out the Cindy Shin template for the project and download some images that best fit the project

-  Creating a Javascript file to design my hamburger bar and make the featured speaker section dynamic

-  Ensuring no linter error

-  Recording a video presenting the features of the project I built. which can be accessed by the link below:
https://www.loom.com/share/e442252a830f41c2b66df8aa12910e05

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+          <div class="copyright">
+            <img src="image/logoimg.jpg" alt="logo" />
+            <p>Web Developers conference</p>
+          </div>
+          <div class="year">
+            <p>&copy; 2023 Web dev conference. All rights Reserved</p>
+          </div>
+        </div>
+      </section>
+    </div>
```
Comment (bellom) [inline index.html]: Let's work on the design to match the [Figma design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) given for this project.

The desktop version looks good 👍, however, we need to adjust the design for mobile.

Figma design:
image

Your design: (kindly remove the partners section on mobile)
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+          <div class="copyright">
+            <img src="image/logoimg.jpg" alt="logo" />
+            <p>Web Developers conference</p>
+          </div>
+          <div class="year">
+            <p>&copy; 2023 Web dev conference. All rights Reserved</p>
+          </div>
+        </div>
+      </section>
+    </div>
```
Comment (bellom) [inline about.html]: For the About page, your mobile design looks good 👍, however, we need to update the design for desktop. Remove the partners section, and set your footer to have a black background.

Figma design Desktop:
image

Your design desktop:
image

**Review by bellom — CHANGES_REQUESTED** [review]
@Megagig,

#### This is a manual review

### Request Changes :+1:

Check my inline comments below, let's make sure your design matches the Figma design provided for this project requirement.

Cheers, and Happy coding!:clap::clap::clap:

**Comment by bellom** [comment]
@Megagig,

#### This is a manual review

### Request Changes :+1:

You have introduced a bug while trying to implement the previous request changes. 

As stated in the previous review, make sure your design matches the Behance design given - [this](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

Once you confirm your design matches this design for both desktop and mobile, let us know.

Cheers, and Happy coding!:clap::clap::clap:

File: index.html (hunk 1 above)
Comment (Megagig) [reply index.html]: @bellom Thanks so much, this has been corrected

File: about.html (hunk 2 above)
Comment (Megagig) [reply about.html]: @bellom Thanks so much for this feedback. This has been corrected.

**Review by bellom — APPROVED** [review]
@Megagig,

### This is a manual review

# Approved :+1:

Cheers, and Happy coding!:clap::clap::clap:

**Pull request merged**